services/utils/dateFormat.js

```diff
@@ -0,0 +1,17 @@
+export default function dateFormat({ format = "long", date }) {
+  let dateTimeStyle = { dateStyle: "short" };
+
+  switch (format) {
+    case "short":
+      dateTimeStyle = { dateStyle: "short" };
+      break;
+    case "long":
+      dateTimeStyle = {
+        dateStyle: "full",
+        timeStyle: "long",
+      };
+      break;
+  }
+
+  return new Intl.DateTimeFormat("en-GB", dateTimeStyle).format(new Date(date));
```

Comment:
This will confuse people as dates will be displayed as `DD/MM/YYYY` everywhere.

Reply:
But isn't that what we have at the moment? I just abstracted the duplicate code

Reply:
I might be, but it shouldn't.

On the server side, there should be a consistent format.
But on the client side, it should use the local format.
For example in components/event/EventCard.js, the code in the `useEffect` should use local format.

With the function this way, there is no option to use local time.
To specify to use local time, the function should look like:
```js
Intl.DateTimeFormat(undefined, options)
```

Reply:
Yep makes sense 👍  I will update